docs/content/users/quickstart.md

```diff
@@ -196,6 +196,18 @@ The legacy type `drupal` will be interpreted as the latest stable version of Dru
     ddev launch $(ddev drush uli)
     ```
 
+=== "Drupal CMS"
+
+    ```bash
+    mkdir my-drupal-cms && cd my-drupal-cms
+    ddev config --project-type=drupal11 --docroot=web
+    ddev composer create --stability="beta" drupal/cms
```

Comment:
```suggestion
    ddev composer create drupal/cms --stability="beta"
```

The only thing I would change would be to place the constraint last, so that if you "arrow up" to previous commands in the CLI history, under `ddev composer create dru...` you would get suggestions for Drupal and Drupal CMS, in the future. Also, I like to order elements from most the least importance, going from left to right.